### PR TITLE
FAQ updates

### PR DESCRIPTION
**Pull request by subhramit**
- Added three new FAQs - wrong gradle tasks, lychee test failing and accidental commit of submodules.
- Fixed typos
- Added a TODO (refs https://github.com/JabRef/jabref/pull/11758).

Partially closes - https://github.com/JabRef/jabref-issue-melting-pot/issues/504

Previews (from online github editor):
image
image
image
image
image

### Mandatory checks

- [ ] ~Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)~
- [ ] ~Tests created for changes (if applicable)~
- [ ] ~Manually tested changed features in running JabRef (always required)~
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by subhramit** [comment]
@koppor (TODO as per discussion) We should now also add an automated test to check submodule diff, and since the text is long, we can just add a link to the submodule section of the FAQ in the bot message (if failing).

**Comment by subhramit** [comment]
I am unsure how to fix the ordered list item prefix failing markdown test. Some points end with a code block, so a surrounding empty lines must be there. But then, the expected prefix resets to "1" for those steps.

File: docs/code-howtos/faq.md [hunk 1]
```diff
@@ -72,4 +76,65 @@ Execute the Gradle task `clean` from the `build` group of the Gradle Tool Window
 
 Execute gradle task `clean` from the `build` group of the Gradle Tool Window in IntelliJ.
 
+### `No test candidates found`
+
+You probably chose the wrong gradle task.
```
Comment (koppor) [inline docs/code-howtos/faq.md:81]: Please add a screenshot

alt text

**Comment by subhramit** [comment]
@koppor Screenshot added. I have also added preview of the added FAQ sections in the PR description (as per github editor preview).

**Review by koppor — APPROVED** [review]

**Pull request merged**